linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/ansible/shared.yml

```diff
@@ -0,0 +1,16 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: Collect users with not correct maximum time period between password changes  
+  command: >
+    awk -F: '$5 > 60 || $5 == "" {print $1}' /etc/shadow
+  register: user_names
+  
+- name: Change the maximum time period between password changes 
+  command: >
+    passwd -x 60 {{ item }}
+  with_items: "{{ user_names.stdout_lines }}"
+  when: user_names.stdout_lines | length > 0
+ 
```

Comment:
Needs a new line at the end of the file.

Reply:
Hello @Mab879 - I had done what you proposed

Reply:
Looks like you have some trailing white-space on that last line. The last line of the file should be an empty (no white space).